vendor/github.com/kubeslice/worker-operator/api/v1beta1/serviceimport_types.go

```diff
@@ -44,6 +44,9 @@ type ServiceImportSpec struct {
 	DNSName string `json:"dnsName"`
 	// Ports which should be exposed through the service
 	Ports []ServicePort `json:"ports"`
+	// Alias names for the exported service. The service could be addressed by the alias names
+	// in addition to the slice.local name.
+	Aliases []string `json:"aliases,omitempty"`
```

Comment:
shouldn't these changes should go in the worker-operator repo and pull in with vendor update?

Reply:
Yes, the go.mod would be updated after we have a tag for the worker-operator repo.